### ISSUE TITLE
Question regards to the training set (mini-imagenet train)

### ISSUE THREAD

**Issue by zilunzhang** [IBM/cdfsl-benchmark#23]
Hi, I visualized some images in the aircraft carrier category and compared them with the original mini-ImageNet dataset provided by matching network in [this website](https://drive.google.com/file/d/1iBu_Iqt49opXHSUNcTRU2WQas1WICLwQ/view). The image id I checked is n02687172_5357.JPEG (from dataset of this competition), which could not be found on the txt file. Then I checked the category name, there are only 4 categories matched from the competition dataset to the list of txt file provided by the matching network. Since the training set is chosen 64 categories from 100 categories of ImageNet, the number of the matched category is unbelievable. I wonder do you use the subset of ImageNet2012 or modified something else?

**Comment by yunhuiguo** [comment]
Hi, our splits follow https://github.com/twitter/meta-learning-lstm/tree/master/data/miniImagenet.

**Comment by zilunzhang** [comment]
> Hi, our splits follow https://github.com/twitter/meta-learning-lstm/tree/master/data/miniImagenet.

But you mentioned in the paper (Section 6) that "For meta-learning methods, we meta-train all meta-learning methods on the base classes of miniImageNet [33]" and [33] is "Matching networks for one shot learning". Those two datasets not completely overlap and the one using twitter repo has few points decrease on accuracy. The protonet result in the paper cannot be a baseline for the competition? Can we use the matching network version mini-imagenet dataset as training set in the competetion?

**Comment by yunhuiguo** [comment]
Hi @zilunzhang , we cite [33] since Matching networks first proposed this dataset. The protonet can be used as a baseline. All the methods should use the provided version mini-imagenet dataset for fairness.

**Comment by zilunzhang** [comment]
> Hi @zilunzhang , we cite [33] since Matching networks first proposed this dataset. The protonet can be used as a baseline. All the methods should use the provided version mini-imagenet dataset for fairness.

Thanks. So, based on my understanding you used twitter split instead of the matching network split in your paper to do all the experiments?

**Comment by yunhuiguo** [comment]
> > Hi @zilunzhang , we cite [33] since Matching networks first proposed this dataset. The protonet can be used as a baseline. All the methods should use the provided version mini-imagenet dataset for fairness.
> 
> Thanks. So, based on my understanding you used twitter split instead of the matching network split in your paper to do all the experiments?

Yes
